src/test/java/uk/ac/ebi/ega/accession/file/FileRestControllerTest.java

```diff
@@ -49,9 +47,9 @@
 
     @Test
     public void testRestApi() {
-        FileDTO fileA = new FileDTO(HashType.MD5, "checksumAAAAAAAAAAAAAAAAAAAAAAAA");
-        FileDTO fileB = new FileDTO(HashType.MD5, "checksumBBBBBBBBBBBBBBBBBBBBBBBB");
-        FileDTO fileC = new FileDTO(HashType.SHA1, "checksumCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC");
+        FileDTO fileA = new FileDTO("checksumAAAAAAAAAAAAAAAAAAAAAAAA", null);
+        FileDTO fileB = new FileDTO("checksumBBBBBBBBBBBBBBBBBBBBBBBB", null);
+        FileDTO fileC = new FileDTO("checksumCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC", null);
```

Comment:
This last checksum was a Sha2 one, you need to put a valid md5 and the sha2 in this case. The test expects that all the elements are correct, valid and that they are gonna be accepted.